pandas/_libs/algos.pyx

```diff
@@ -230,14 +232,15 @@ def kth_smallest(numeric[:] a, Py_ssize_t k) -> numeric:
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def nancorr(ndarray[float64_t, ndim=2] mat, bint cov=0, minp=None):
+def nancorr(float64_t[:, :] mat, bint cov=0, minp=None, bool parallel=False):
```

Comment:
+1 on changing to memoryviews where viable.  if we don't alter `mat` inplace, might want to add the `const` modifier.

Reply:
:+1: